spec/string-functions/quote.hrx

```diff
@@ -0,0 +1,80 @@
+<===> unquoted/input.scss
+a {
+  a: quote(a);
+}
+
+<===> unquoted/output.css
+a {
+  a: "a";
+}
+
+<===>
+================================================================================
+<===> quoted-double/input.scss
```

Comment:
Style nit: paths should be [underscore-separated](https://github.com/sass/sass-spec/blob/master/STYLE_GUIDE.md#prefer-underscore-separated-paths).

Reply:
Done